libs/partners/pinecone/langchain_pinecone/vectorstores.py

```diff
@@ -74,6 +74,7 @@ class PineconeVectorStore(VectorStore):
                     dimension=1536,
                     metric="cosine",
                     spec=ServerlessSpec(cloud="aws", region="us-east-1"),
+                    deletion_protection="disabled",  # Defaults to "enabled"
```

Comment:
The default behavior is `deletion_protection="disabled"` So this comment is incorrect.

Reply:
Eek, silly oversight. Thank you!